examples/custom_code_deployment/README.md

```diff
@@ -188,6 +188,15 @@ zenml artifact-store register gcs_store -f gcp \
     --authentication_secret=gcp_secret
 ```
 
+Note that you can deploy your GCS Store using the ZenML CLI as well, using the
+`zenml artifact-store deploy` command. This is how you would do it:
+
+```shell
+zenml artifact-store deploy gcs_store --flavor=gcs --project_id=my_project ...
+```
+
+For more information on this `deploy` subcommand, please refer to the 
+[documentation](https://docs.zenml.io/advanced-guide/practical-mlops/stack-recipes#deploying-stack-components-directly).
```

Comment:
In the live 0.37.0 version I don't see any documentation for the `zenml ... deploy` command on that page. Is that done on some other branch perhaps?